### ISSUE TITLE
Tarren mill deathguards and southshore guards wrong level

### ISSUE THREAD

**Issue by ghost** [Wall-core/Everlook-Bugtracker#433]
16 years ago they were level 45, I remember distinctively.

**Comment by Tessablanc** [comment]
Pretty sure this is correct--level 55 and 60 guards were restricted to major cities and starting areas. The idea is that once you're a big enough boy to play around in Hillsbrad, you don't need max-level guards protecting you any more.

**Comment by ghost** [comment]
This is not correct and I can confirm this https://www.wowhead.com/tbc/npc=2405/tarren-mill-deathguard
They were originally level 45 from the years of 2004-2006 until the Pre-TBC patch was introduced.  
https://www.gamermaps.net/world-of-warcraft/map/npc/Tarren+Mill+Deathguard
In this Classic WoW Blizz Post I am not alone in this link here >>https://eu.forums.blizzard.com/en/wow/t/southshore-and-tarrens-mill-guards-are-off/81734/7
Quote"_The new project lead wanted to kill-off spontaneous world-PvP ASAP. They wanted everyone raiding or in BGs, completely missing the point of the world that Mark Kern oversaw being created. That meant making both TM and SS virtually impenetrable._"

They were level 45 with over 3,500 HP.
They spawn randomly when you attack as well.

**Comment by Tessablanc** [comment]
By "this is correct," I meant your post. They should be level 45.

**Comment by Koveflash** [comment]
Please.

**Comment by ghost** [comment]
Any chance on this happening

**Comment by Tessablanc** [comment]
I found some SS/TM pvp footage from 2005: https://youtu.be/5hc4Ap50Zrg?t=100

The guards were actually level 50!

And here's footage from Classic, where they made the guards level 55: https://youtu.be/yL7VlT_Y2FE?t=50

This is a perfect example of how Classic is a fake, inaccurate version of vanilla and shouldn't be used as a reference for anything. Let's have level 50 Southshore Watchmen like real 2005 WoW, not level 55 guards like a fake corporate private server from 2019.

**Comment by ghost** [comment]
I believe southshore guards were level 50 due to less guards being there.
I remember wondering why tarren mill used lvl 45 guards when southshore had 50.

**Comment by Tessablanc** [comment]
I can believe that, because TM is situated in a higher level part of the zone, and generally the level of town guards scales inversely with the level of the surrounding content. It would be nice to see some proof, though.

Classicdb.ch lists Deathguards at level 50 and Southshore guards at level 55, so it's likely both types are just 5 levels too high.

Edit: Here's some ancient archived data from Allakhazam. It says Southshore guards are 50, but says TM guards are "47-50."

https://web.archive.org/web/20050217110700/http://wow.allakhazam.com/db/mob.html?wmob=2386

https://web.archive.org/web/20050313132944/http://wow.allakhazam.com/db/mob.html?wmob=2405

The maps showing TM Deathguard locations are pretty funny!

**Comment by ghost** [comment]
Yes I remember the guards in TM spawning at different level variables and being level 50 was one of them, I even remember seeing a level 48

**Comment by Tessablanc** [comment]
Also interesting that the TM guards were apparently coded as undead, rather than humanoid. This appears to be true of the Undercity guards as well, despite most of the Forsaken NPCs there being humanoid: https://web.archive.org/web/20050904073446/http://wow.allakhazam.com/dyn/mobs/zone7.html

**Comment by ghost** [comment]
yes they should be undead as priests use to be able to cast shackle undead on them during battles at TM.

**Comment by ghost** [comment]
has this been corrected?

**Comment by Daribon** [comment]
> has this been corrected?

No, but I have a way more detailed bug report on vmangos tracker:
https://github.com/vmangos/core/issues/1730
